pkg/dsl/cst/builtin_functions.go

```diff
@@ -78,7 +79,7 @@ func (root *RootNode) BuildMultipleArityFunctionCallsiteNode(
 		return root.BuildTernaryFunctionCallsiteNode(astNode, builtinFunctionInfo)
 	}
 
-	return nil, fmt.Errorf(
+	return nil, errors.New(
 		"at CST BuildMultipleArityFunctionCallsiteNode: function name not found: " +
 			builtinFunctionInfo.name,
 	)
```

Comment:
You could make this actually format instead of a concatenation:
```suggestion
	return nil, fmt.Errorf(
		"at CST BuildMultipleArityFunctionCallsiteNode: function name not found: %s",
		builtinFunctionInfo.name,
	)
```